client/src/templates/Challenges/components/interactive-editor.tsx

```diff
@@ -79,18 +86,19 @@ const InteractiveEditor = ({ files }: Props) => {
           },
           syntax: freeCodeCampDarkSyntax
         }}
-        options={{
-          editorHeight: 450,
-          editorWidthPercentage: 60,
-          showConsole: showConsole,
-          showConsoleButton: showConsole,
-          layout: got('html') ? 'preview' : 'console',
-          showLineNumbers: true
-        }}
-      />
+      >
+        <SandpackLayout>
+          <MonacoEditor />
+          <SandpackPreview
+            style={{
+              display: isHtmlOnly ? 'flex' : 'none'
+            }}
+          />
```

Comment:
```suggestion
          {got('html') && <SandpackPreview />}
```
Again, no need to change the logic. Also it seems wasteful to render a component only to hide it.